unikernel.ml

```diff
@@ -301,6 +301,21 @@ struct
               reply);
         []
 
+  let single_unikernel albatross ~user_name ~unikernel_name =
+    Albatross.query albatross ~domain:user_name ~name:unikernel_name
+      (`Unikernel_cmd `Unikernel_info)
+    >|= function
+    | Error msg ->
+        Logs.err (fun m -> m "error while communicating with albatross: %s" msg);
+        []
+    | Ok (_hdr, `Success (`Unikernel_info unikernels)) -> unikernels
```

Comment:
What about directly here constraining to a single unikernel... and return `(Name.t * Unikernel.info) option` instead of having to deal with that at the call sites?